app/models/course/condition/achievement.rb

```diff
@@ -18,6 +18,11 @@ def satisfied_by?(course_user)
     course_user.achievements.exists?(achievement)
   end
 
+  # Array of classes that the condition depends on.
+  def self.dependent_classes
```

Comment:
Class methods are methods in the class' singleton class.

So, to do that, define another module (`ClassMethods` by convention, so `ConditionClassMethods`) then `extend ClassMethods` after the `include ConditionInstanceMethods` line.